web-api/src/business/useCaseHelper/service/createChangeItems.ts

```diff
@@ -66,7 +66,8 @@ const createDocketEntryForChange = async ({
 
   const isAddressSealed =
     caseEntity.petitioners.find(
-      petitioner => petitioner.email === newData.email,
+      petitioner =>
+        petitioner.email === newData.email && petitioner.name === newData.name,
```

Comment:
Mostly out of curiosity, what prompted this change? If there's a chance of ambiguity here, but should probably compare by `userId`, even if it means passing more data around